### ISSUE TITLE
<bug brief>error when delegating

### ISSUE THREAD

**Issue by leonelb11** [nutbox-dao/peanut-interface-v2#11]
<bug details>

Hi, my steemit username is @leonelb. Today I made a delegation of 200 sp to nutbox and it is still not reflected on the page.

![image](https://user-images.githubusercontent.com/80784330/111369853-e6670300-866d-11eb-97dd-e1fc701acb26.png)

![image](https://user-images.githubusercontent.com/80784330/111370353-7f961980-866e-11eb-9905-4908258d2faf.png)

delegation test.

![image](https://user-images.githubusercontent.com/80784330/111370551-c257f180-866e-11eb-8a63-f9fead49d43b.png)

If I refresh the page, it appears that I have 0 pnut and then the amount mined so far is reflected, however, it is at a standstill.

address:TV1HUGWuFycuugcoaasVZiBJ9Mktf4weoB

name in steemit: @leonelb

**Comment by terryyyyyy** [comment]
@leonelb11 Hi, can input 201 sp for try?

**Comment by leonelb11** [comment]
ok, no problem.

**Comment by leonelb11** [comment]
hello, I tried to place 201 pnut, but an error occurred. The error says:
"Delegation error, insufficient rate".

I leave you the images so you can see how it was.

Thank you very much for your effort.

**Comment by terryyyyyy** [comment]
@leonelb11 So, where is the pic?

**Comment by leonelb11** [comment]
I attached three images, can't you see them?

**Comment by terryyyyyy** [comment]
Can you show me the error page ?

**Comment by leonelb11** [comment]
Sorry @terryyyyyyyy, I was replying via gmail and have been attaching the images from gmail. At no time did I think you could not view them. Please excuse me if I wasted your time. These are the images of the whole process:

Image 1 - Steem available
![steem available - image 1](https://user-images.githubusercontent.com/80784330/111771481-24d50b80-8882-11eb-80e9-fe09c04bc918.png)

Image - 201 Sp

![201 SP - image 2](https://user-images.githubusercontent.com/80784330/111771573-40401680-8882-11eb-9d2b-490c859f47de.png)

Image 3 - Delegation Error

![Delegation error - image 3](https://user-images.githubusercontent.com/80784330/111771668-56e66d80-8882-11eb-9c4d-4147902493dd.png)

It tells me: Delegation error, insufficient tariff.

Thank you very much for your work. 

Sincerely, I apologize for the inconvenience.

**Comment by terryyyyyy** [comment]
@leonelb11 I see it, it shows your delegation operate blocked because of insufficient fee.The delegation operation cost 1 steem.It shows here:
![image](https://user-images.githubusercontent.com/15071907/111802647-31e90d80-8909-11eb-8365-87784f91bc93.png)

**Comment by leonelb11** [comment]
Hi @terryyyyyy. At the time I did the delegation, I am sure I had 1 steem and I got the error. Now I have tried again and it came up fine, however, it tells me that I have 201 sp mining, which is not true. I should have 700 sp mining. 

![image](https://user-images.githubusercontent.com/80784330/111806118-0b46ba80-88a8-11eb-85a1-7ad49078338a.png)

![image](https://user-images.githubusercontent.com/80784330/111806183-1a2d6d00-88a8-11eb-896f-370f05445fbb.png)

![image](https://user-images.githubusercontent.com/80784330/111806197-1ef22100-88a8-11eb-809d-1056b7f7e367.png)

![image](https://user-images.githubusercontent.com/80784330/111806216-24e80200-88a8-11eb-88ea-d696bed14ae4.png)

Thank you very much for your work, sorry for the inconvenience.
